.github/workflows/windows_job.yml

```diff
@@ -126,12 +135,20 @@ jobs:
             echo "/c/Jenkins/Miniconda3/python.exe -m pip install --upgrade pywin32==300"
             # Source conda so it's available to the script environment
             echo "source /c/Jenkins/Miniconda3/etc/profile.d/conda.sh";
-            echo "${SCRIPT}";
-          } > "${RUNNER_TEMP}/exec_script"
+            echo 'export PATH="/c/Jenkins/Miniconda3/Scripts:${PATH}"'
```

Comment:
Driveby for https://github.com/pytorch/vision/blob/14553fb9eb78e053100e966feee6149905f9922c/.github/workflows/test-windows.yml#L41-L42